ArkGameExample/TankGame/TankGameManager.swift

```diff
@@ -79,6 +80,7 @@ class TankGameManager {
 //                    // center of the camera's output in screen coordinates, relative to screen
 //                    screenPosition: CGPoint(x: screenWidth / 2, y: screenHeight / 2),
 //                    // relative to screen
+//                    screenSize: CGSize(width: canvasWidth, height: canvasHeight),
```

Comment:
i think we would want to use `screenSize` for screenSize and not canvasWidth, canvasHeight